trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/model/ServerInfoResponse.java

```diff
@@ -0,0 +1,77 @@
+/**********************************************************************
+ * Copyright (c) 2024 Ericsson
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License 2.0 which
+ * accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+package org.eclipse.tracecompass.incubator.internal.trace.server.jersey.rest.core.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+/**
+ * Contributes to the model used for TSP swagger-core annotations.
+ *
+ * @author Vlad Arama
+ */
+@Schema(description = "System Information Response")
+public interface ServerInfoResponse {
+
+    /**
+     * @return Server version in the format Major.Minor.Micro
+     */
+    @Schema(required = true, description = "Server version in the format Major.Minor.Micro")
+    String getServerVersion();
+
+    /**
+     * @return Build time or qualifier of the server version, if available
+     */
+    @Schema(description = "Build time or qualifier of the server version, if available")
+    String getBuildTime();
+
+    /**
+     * @return Operating system name
+     */
+    @Schema(required = true, description = "Operating system name")
```

Comment:
FYI. I get a deprecated warning. `required` is deprecation. However, it's ok to keep it. We'll update it with the others.